doc/developer/index.adoc

```diff
@@ -281,16 +301,14 @@ The SDK and onejar archives are hosted on https://www.eclipse.org/downloads/ . T
 
 === Building the SDK and onejar
 
-1. Check out the release tag
-2. Enter the `core` directory:
-
-   cd core
-
-3. From the `core` directory, execute the following:
+1. Check out the release tag of https://github.com/eclipse/rdf4j[rdf4j]
+2. From the root directory, execute the following:
 
-   mvn -Passembly clean package
+   mvn -Passembly clean install -DskipTests
```

Comment:
Slight optimization: perhaps make the skipTests flag part of the assembly profile, so that you don't have to specify both on the command line?

Reply:
There is no harm in omitting -DskipTests (it just takes long). I believe the current assembly profile also runs the tests unless otherwise directed. I'm not sure assembly could be as efficient if skipTests is not included on the command line. I have a profile (called compliance) that activates when the skipTests is omitted, but I think skipTests must be on the command line to prevent the compliance profile from activating.